[MWPW-136492] Enable Locale-Specific Subtitles on Embedded Vimeo

toka1300 opened the pull request:
Sets the subtitles of embedded Vimeo videos based on the user's locale. This is only enabled for English, Japanese, French and German.

Testing: Click on one of the "Play Video" buttons, the stage link will not have any subtitles chosen by default, whereas the branch link will have French.

Resolves: [MWPW-136492](https://jira.corp.adobe.com/browse/MWPW-136492)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/fr/drafts/embed-video-subtitles
- After: https://mwpw-135726-embed--express--adobecom.hlx.page/fr/drafts/embed-video-subtitles

qiyundai commented:
I realize the getLocale is going to return 'us' for our en pages, which isn't a valid texttrack for vimeo. Can you make a us -> en swap or maybe chain a getLanguage and test the result of that?

toka1300 commented:
> I realize the getLocale is going to return 'us' for our en pages, which isn't a valid texttrack for vimeo. Can you make a us -> en swap or maybe chain a getLanguage and test the result of that?

Fixed this and updated languages available.
